volontulo_org/settings/dev_vagrant.py

```diff
@@ -0,0 +1,36 @@
+"""
+Vagrant Development Settings Module
+"""
+
+from .base import *
+
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+DEBUG_TOOLBAR_PATCH_SETTINGS = False
+
+INSTALLED_APPS += (
+    'debug_toolbar',
+    'django_coverage',
+    'django_extensions',
+    'django_nose'
+)
+
+TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
+
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'volontulo',
+        'USER': 'volontulo',
+        'PASSWORD': 'volontulo',
+        'HOST': '127.0.0.1',
+        'PORT': '5432',
+    }
+}
+
+EMAIL_BACKEND = 'django.core.mail.backends.filebased.EmailBackend'
```

Comment:
duplicate.

EDIT: ok, I see from where You have taken that. Could You, please fix this also in `dev.py` and make `dev_docker.py` coherent with them? Or even create `dev_base.py` and import from it into `dev.py`, `dev_docker.py` and `dev_vagrant.py`, leaving in them only DB settings?